tests/integration/utils.sh

```diff
@@ -67,7 +102,10 @@ function setup_container_environment()
     image_exists "${container_img}"
     if [[ "$?" -ne 0 ]]; then
       current_step=$((current_step + 1))
-      say "[${current_step}/${total_steps}] Building container image for ${distro}. This might take a while..."
+
+      if [[ "$verbose" -eq 1 ]]; then
+        say "[${current_step}/${total_steps}] Building container image for ${distro}. This might take a while..."
```

Comment:
Same question as above. Could you please clarify why you added this again, considering my commit already covers this?